packages/blocks/src/api/registration.js

```diff
@@ -118,7 +119,7 @@ import { DEPRECATED_ENTRY_KEYS } from './constants';
  * @type {Object}
  */
 export const DEFAULT_BLOCK_TYPE_SETTINGS = {
-	icon: 'block-default',
+	icon: blockIcon,
```

Comment:
I think "block-default" is actually most descriptive here. It's the icon a block has until you give it a better one.